Buttons.d

```diff
@@ -326,6 +305,29 @@ func int Button_GetCaptionPtr(var int hndl) {
 	return v.textLines_next;
 };
 
+func void _Button_processMouseEvents( var int btn_hndl ) {
+	var _Button btn; btn = get( btn_hndl );
```

Comment:
Remove white space between parentheses for consistent/readable code

```D
	var _Button btn; btn = get(btn_hndl);
```

Reply:
only accept consistency as reason but not readability, think everybody has their own preference. ;)